dev-docs/bidders/addigi.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: bidder
+title: Addigi
+description: Addigi Bidder Adapter
+biddercode: addigi
+aliasCode : smarthub
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+dchain_supported: true
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+floors_supported: true
+fpd_supported: false
+pbjs: true
+pbs: false
```

Comment:
You may set pbs to true now. The alias PR is approved.

Reply:
ok, pbs setted to true